netflix/src/components/main/sort/SortBy.tsx

```diff
@@ -12,16 +13,18 @@ const SortBy = () => {
   }, [open]);
   return (
     <div className="sortBy">
-      <h1 style={{ color: "#555555" }}>Sort By</h1>
+      <h1 style={{ color: "#555555", paddingRight: "5px" }}>Sort By: </h1>
       <div style={{ display: "inline-flex" }}>
-        <h1 onClick={handleOpen}>Release Year</h1>
-        <AiFillCaretDown />
+        <h1 onClick={handleOpen}>
+          Release Year <AiFillCaretDown className="caret" />
```

Comment:
You should technically show current selected sort option on the screen. Right now you always show `release year`